deploy/resources/v0.8.0/02-clustertasks/s2i-go-task.yaml

```diff
@@ -0,0 +1,59 @@
+# auto generated by script/update-tasks.sh
+# DO NOT EDIT: use the script instead
+# source: https://raw.githubusercontent.com/openshift/pipelines-catalog/release-v0.7/s2i-go/s2i-go-task.yaml
+#
+---
+apiVersion: tekton.dev/v1alpha1
+kind: ClusterTask
+metadata:
+  name: s2i-go
+spec:
+  inputs:
+    resources:
+      - name: source
+        type: git
+    params:
+      - name: PATH_CONTEXT
+        description: The location of the path to run s2i from.
+        default: .
+        type: string
+      - name: TLSVERIFY
+        description: Verify the TLS on the registry endpoint (for push/pull to a non-TLS registry)
+        default: "true"
+        type: string
+  outputs:
+    resources:
+      - name: image
+        type: image
+  steps:
+    - name: generate
+      image: quay.io/openshift-pipeline/s2i
```

Comment:
Shouldn't we use tags for those ?

Reply:
Did we ever do the release of the images? I was hoping if we did,

Reply:
yes, it is a very good idea to pin down versions.
An issue against catalog?

Reply:
We have to do quay image mirroring thing change as well to get them pinned,